pkg/router/controller/status_test.go

```diff
@@ -428,3 +428,41 @@ func TestFindOrCreateIngress(t *testing.T) {
 	}
 	t.Logf("routes: %#v", route.Status.Ingress)
 }
+
+func TestLastTransitionTimeIsRFC3339(t *testing.T) {
+	p := &fakePlugin{}
+	c := testclient.NewSimpleFake(&routeapi.Route{})
+	admitter := NewStatusAdmitter(p, c, "test")
+
+	route := routeapi.Route{
+		ObjectMeta: kapi.ObjectMeta{Name: "route1", Namespace: "default", UID: types.UID("uid1")},
+		Spec:       routeapi.RouteSpec{Host: "route1.test.local"},
+	}
+
+	// Add the route and validate that the status is set.
+	err := admitter.HandleRoute(watch.Added, &route)
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	if len(c.Actions()) != 1 {
+		t.Fatalf("unexpected actions: %#v", c.Actions())
+	}
+	action := c.Actions()[0]
+	if action.GetVerb() != "update" || action.GetResource() != "routes" || action.GetSubresource() != "status" {
+		t.Fatalf("unexpected action: %#v", action)
+	}
+	obj1 := c.Actions()[0].(ktestclient.UpdateAction).GetObject().(*routeapi.Route)
+	condition1 := obj1.Status.Ingress[0].Conditions[0]
+	timestamp := condition1.LastTransitionTime
+	if timestamp == nil || condition1.Status != kapi.ConditionTrue || condition1.Reason != "" {
```

Comment:
This test is a copy-paste job and the conventions it follows are consistent with every single other test in this file.  But rather than argue about conventioln, I'm going to strip this test down to the only thing I actually care about testing (the timestamp format).